src/message.ts

```diff
@@ -30,3 +30,9 @@ export function fromContent(contents: string): Message {
       };
   }
 }
+
+export function toString(message: Message): string {
```

Comment:
We have nice unit tests on this module, so I could add a test for this. It seemed a bit too trivial to do that and then leave the slack-notification itself untested (since I have no idea how to do that). :shrug: